.github/workflows/release.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         env:
           SIGNING_CERT: ${{ secrets.WINDOWS_SIGNING_CERT }}
         run: |
-          Invoke-WebRequest https://sourceforge.net/projects/nsis/files/NSIS%203/3.06.1/nsis-3.06.1-strlen_8192.zip/download?use_mirror=excellmedia -UseBasicParsing -Outfile ./nsis.zip
+          Invoke-WebRequest "https://sourceforge.net/projects/nsis/files/NSIS%203/3.06.1/nsis-3.06.1.zip/download" -UseBasicParsing -UserAgent [Microsoft.PowerShell.Commands.PSUserAgent]::Chrome -Outfile "./nsis.zip"
```

Comment:
This will cause problems. We need to large strings build of NSIS (strlen 8192)
Refer  https://github.com/getgauge/gauge/issues/150

Reply:
Have updated the build uri... dint realise they were different builds